sdk/src/main/scala/io/horizen/account/state/ForgerStakeMsgProcessor.scala

```diff
@@ -220,6 +220,20 @@ case class ForgerStakeMsgProcessor(params: NetworkParams) extends NativeSmartCon
     stakeList
   }
 
+  def getListOfForgersStakesOfUser(view: BaseAccountStateView, owner: Address): Seq[AccountForgingStakeInfo] = {
```

Comment:
Consider using just one implementation for both this method and `getListOfForgersStakes` above, for instance using an optional input parameter `owner: Option[Address]`

Reply:
Done